webvtt.go

```diff
@@ -38,6 +39,35 @@ func parseDurationWebVTT(i string) (time.Duration, error) {
 	return parseDuration(i, ".", 3)
 }
 
+// https://tools.ietf.org/html/rfc8216#section-3.5
+func computeOffsetFromTimestampMap(line string) (time.Duration, error) {
+	var timeOffset time.Duration
+
+	right := strings.Split(line, "=")[1]
```

Comment:
You should check whether `len(strings.Split(line, "=")) > 0` otherwise it may panic here